nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard
# Sample Data Update

## Description
The sample data is updated to include the current equipment data in Firebase

## Related Issue
https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/issues/72

## Motivation and Context
The app should have initial data available for a good rendering of the site.
The sample data give us a backup if the Firebase data is lost.

## How Can This Be Tested?
Check for files changes. Check that the sample data reflects the desired data for the initial app load.

## Types of changes
